### DISCUSSION TITLE
New language supporting

### DISCUSSION THREAD

**Discussion by itbeard** [mastodon/mastodon#22015] [Q&A]
Hello folks,
I'm presented Belarusian Masto server vkl.world.
Two weeks ago we initialized the translation Mastodone to the Belarusian language and now it is done for 80% (https://crowdin.com/project/mastodon/be).

My question: how usually long does it take to have a new official build of Mastodone with supporting languages from Crowdin (now we are using 4.0.2)? 

Thx in advance!

**Comment by playmobilmeister** [comment]
Unfortunately, many months seems to be the norm. Our last major update to the Hebrew UI took almost 6 months to be pulled into Mastodon. [This profile](https://crowdin.com/profile/ykzts) seems to be responsible for adding new languages but I am not sure whether they are also merging them.
